types/block.go

```diff
@@ -141,7 +141,7 @@ func (b *Block) IsZero() bool {
 
 // ChainID returns chain ID of the block.
 func (b *Block) ChainID() string {
-	return b.SignedHeader.ChainID() + "-block"
```

Comment:
not sure if this will work? as you are setting `-block` still

Reply:
i think chainId should also use `syncType`?

Reply:
what about this @tzdybal ?

Reply:
This works fine as shown in follow-up PR: https://github.com/rollkit/rollkit/pull/1614/files#diff-7af4c73284d8c84344fe9b93472357f0318453ef83c0c30370343856b2d5a985R652

It enforces P2P only sync (without DA). And test and logs clearly shows that blocks are synced correctly without errors. `-block` is added only on go-header layer, and this is sufficient.